src/selectors/toySelectors.js

```diff
@@ -1,2 +1,2 @@
 export const selectToys = ({toys}) => toys;
-export const selectToy = (toyId) => ({toys}) => toys.find(toy => toy.id === toyId);
+export const selectToy = (toyId) => ({toys}) => toys.find(toy => toy.id === parseInt(toyId));
```

Comment:
Is the toy.id stored as a string or a number?

If it's a number, then you'll want to fix the toySelectors.test.js (it will fail with this parseInt() added) and you'll want to update the fixtures/toys.js so it reflects a number for the id.

Reply:
Assuming the ID comes from Rails, then this is definitely a number, so you'll want to update selector test and fixtures to reflect that it's an ID instead of a string.

Reply:
```
 FAIL  src/test/unit/selectors/toySelectors.test.js
  toySelectors
    selectToys
      ✓ returns all the toys (2 ms)
    selectToy
      ✕ returns the appropriate toy (3 ms)

  ● toySelectors › selectToy › returns the appropriate toy

    expect(received).toEqual(expected) // deep equality

    Expected: {"description": "Sample Toy #1 description", "id": "1", "image_url": "https://m.media-amazon.com/images/I/61DWaMHr8HL._AC_SX425_.jpg", "name": "Sample Toy #1", "reviews": [{"content": "Review content!", "id": 1001, "rating": "5", "title": "Review title!"}]}
    Received: undefined

      17 |     
      18 |     it('returns the appropriate toy', () => {
    > 19 |       expect(selector(toyWithReview.id)).toEqual(toyWithReview);  
         |                                          ^
      20 |     });
      21 |   });
      22 | });

      at Object.<anonymous> (unit/selectors/toySelectors.test.js:19:42)

Test Suites: 1 failed, 2 passed, 3 total
Tests:       1 failed, 11 passed, 12 total
Snapshots:   0 total
Time:        2.024 s

```

Reply:
Okay, thank you! I just realized I guess I could've fixed it on this same branch instead of making another pull request, right???

Reply:
@Halezmarie You definitely shouldn't not be merging in bad code.

Reply:
> Okay, thank you! I just realized I guess I could've fixed it on this same branch instead of making another pull request, right???

Correct, you'll want to make these updates to the same branch, so they show up as changes in the same PR.  The reviewer will see there are new commits and can filter to see the new changes to make re-reviewing easy.  If you make changes outside the PR, you lose the review history, etc. so you definitely want to be committing back to same branch so it shows up in the PR.

Then, after back and forths and PR gets approved, you can go ahead and merge it in.